fluentui_core/src/androidTest/java/com/microsoft/fluentui/core/ExampleInstrumentedTest.kt

```diff
@@ -0,0 +1,24 @@
+package com.microsoft.fluentui.core
```

Comment:
Please remove these extra test files/directories which are not being used

Reply:
Done